django/contrib/auth/context_processors.py

```diff
@@ -25,6 +25,9 @@ class PermWrapper:
     def __init__(self, user):
         self.user = user
 
+    def __repr__(self):
+        return f'{self.__class__.__qualname__} user={self.user!r}'
```

Comment:
Kwargs notation is not necessary. Also parentheses are missing:
```suggestion
        return f'{self.__class__.__name__}({self.user!r})'
```

Reply:
isn't `qualname` better than `name`?

Reply:
It's a top-level class internal class so it doesn't matter.